.github/workflows/fossa.yml

```diff
@@ -0,0 +1,35 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Is the purpose of running this towards PRs to validate the dependencies that they add?